File: ProjetoPOO/Models/Agendamento.cs
using System;

namespace ProjetoPOO.Models
{
    class Agendamento : ModeloId
    {
        public string Data { get; set; }
        public int IdCliente { get; set; }
        public int IdPet { get; set; }
        public int IdServico { get; set; }
        public int IdFuncionario { get; set; }

        public Agendamento()
        {
            
        }
        public Agendamento(int id, string data, int idCliente, int idpet, int idServico, int idFuncionario)
        {
            Id = id;
            Data = data;
            IdCliente = idCliente;
            IdPet = idpet;
            IdServico = idServico;
            IdFuncionario = idFuncionario;
        }

        public override string ToString()
        {
            return $"ID: {Id} | Data: {Data:dd/MM/yyyy HH:mm} | Cliente: {IdCliente} | Pet: {IdPet} | Serviço: {IdServico} | Funcionario: {IdFuncionario}";
        }

    }
}

File: ProjetoPOOPython/Models/agendamento.py
import datetime
from Models.modeloId import ModeloId

class Agendamento(ModeloId):
    def __init__(self, id: int = 0, data: datetime = None, id_cliente: int = 0, id_pet: int = 0, id_servico: int = 0, id_funcionario: int = 0):
        super().__init__(id)
        self.data = data if data else datetime.now()
        self.id_cliente = id_cliente
        self.id_pet = id_pet
        self.id_servico = id_servico
        self.id_funcionario = id_funcionario

    def __str__(self):
        return f"ID: {self.id} | Data: {self.data.strftime('%d/%m/%Y %H:%M')} | Cliente: {self.id_cliente} | Pet: {self.id_pet} | Serviço: {self.id_servico} | Funcionário: {self.id_funcionario}"
